### ISSUE TITLE
tags have no language associated with them

### ISSUE THREAD

**Issue by andy5995** [mhwkb/mhwkb.github.io#112]
For example, "e-health" is the same in EN and NL. If the tag is clicked, the post comes up twice.

One instance from the EN category, and one from the NL.

https://mhwkb.github.io/tag/e-health/

**Comment by escottalexander** [comment]
What are possible solutions to this? Does someone need to change a Jekyll setting to require attaching a language to each individual tag?

**Comment by erichoog** [comment]
I think since the tags are associated to a post and a post has a language category we should be able to associate the tags language but not 100% how just yet.  But I'm sure it can be done.

**Comment by andy5995** [comment]
@erichoog wrote in #132 

>This may solve the issue for #112

I was just looking at this more closely and I saw that links from other languages show up. By clicking on the Nederlands language section -> "training" tag, I got results from the English section as well.

https://mhwkb.github.io/tag/training/

But hey.. thanks for trying! :)

**Comment by erichoog** [comment]
Oh so there's a training in English and Dutch??

**Comment by erichoog** [comment]
Upon looking at that post in Dutch (for Vrendle) it looks like they had entered the tags in English, so if we translated them then it wouldn't be an issue.  

This does raise a good point that if tags in two different languages have the same spelling then they would end up on the same tag page.

**Comment by andy5995** [comment]
>it looks like they had entered the tags in English, so if we translated them then it wouldn't be an issue.

Those were entered by @sexybiggetje , a native Dutch speaker. I seem to recall asking him about it. But I forget what he told me. Martijn, can you elaborate on this a little?

**Comment by andy5995** [comment]
>I seem to recall asking him about it. But I forget what he told me.

I accidentally found the conversation on Slack...

https://mhwkb.slack.com/messages/C7HKLQSTT/convo/C7HKLQSTT-1511417733.000181/

>andy5995 3 months ago
The Dutch translation for "training" and "e-health" is the same as English?
Martijn de Boer 3 months ago
Correct, they are [loanwords](https://en.wikipedia.org/wiki/Loanword)
Martijn de Boer 3 months ago
Same as therapie/therapy, but we Dutchiefied that

**Comment by sexybiggetje** [comment]
Yes that's correct. Most language have a form of loanwords. Since it's close relation to English, Dutch has quite a few. We borrow from English and French a lot. And thus Training and E-Health are straight rip offs :). In the case of Therapie/Therapy it's not an issue because the spelling is different.

This is also true for for example Japanese, words that they can't express are borrowed. 
Did you know that it's estimated that English itself is founded on about 1% of loanwords with Dutch origins? #languagegeekstatistics :)

**Comment by erichoog** [comment]
Very cool I didn't realize this! I love the Dutch lol all of my grandparents are from Het Nederlands but my grandparents never really taught me alot as a kid so I only know a tiny bit mostly cuss words lol.

I'm very curious though, when I google "training in Dutch" I get "opleiding" so is that not the direct translation?  Or is it just not commonly used anymore?

**Comment by sexybiggetje** [comment]
Training usually implies short education, as opleiding usually is a multi year education.
It can also translate to exercise.
